server/views/applications/pages/layout.njk

```diff
@@ -45,7 +45,7 @@
             text: "Save and continue"
             }) }}
 
-            <a href="{{ paths.applications.show({ id: applicationId }) }}">Back to task list</a>
+            <a href="{{ paths.applications.show({ id: applicationId }) }}">Exit without saving</a>
```

Comment:
FYI this link appears on all application pages and will be changed across the app. Ideally this would be it's own commit along with changes to `concerns.njk` and `check-your-answers.njk` where this wording is also referenced.